tests/index.js

```diff
@@ -153,12 +184,55 @@ describe('awsManager', async () => {
     })
   })
 
+  describe('getSharedConfigByService', async () => {
+    const getSharedConfigParams = {
+      ASSESSMENT_CONFIG: { name: 'ASSESSMENT_CONFIG', value: { testId: 123459, active: false }, service: 'resolvers', type: 'object' },
+      USER_TABLE: { name: 'USER_TABLE', value: 'table', service: 'resolvers', type: 'string' }
+    }
+    before(function () {
+      AWS.mock('DynamoDB.DocumentClient', 'query', function (params, callback) {
+        callback(null, {
+          Items: [
+            { name: 'ASSESSMENT_CONFIG', service: 'resolvers', value: { testId: 123459, active: false } },
+            { name: 'USER_TABLE', service: 'resolvers', value: 'table' }
+          ],
+          Count: 2,
+          ScannedCount: 2
+        })
+      })
+    })
+    it('should get all the parameters records by service stored in Config_Table', async () => {
+      const params = await awsManager.getSharedConfigByService(env, shareConfigService)
+      console.log(params)
+      for (let i = 0; i < getSharedConfigParams.length; i++) {
+        const found = params[getSharedConfigParams[i].key]
+
+        should.exist(found)
+        found.value.should.equal(getSharedConfigParams[i].value)
+      }
+    })
+
+    it('should return cached parameters records by service stored in Config_Table', async () => {
+      const params = await awsManager.getSharedConfigByService(env, shareConfigService)
+      console.log(params)
+      for (let i = 0; i < getSharedConfigParams.length; i++) {
+        const found = params[getSharedConfigParams[i].key]
+
+        should.exist(found)
+        found.value.should.equal(getSharedConfigParams[i].value)
+      }
+    })
```

Comment:
Test is same as the one before - to test if cached value returned, you would need to first retrieve the values, then update them - and then retrieve them again to verify that you are NOT getting the updated value... thus proving that it is indeed read from cache.

However, you don't have to write such a test right now. So you can remove this one.